### DISCUSSION TITLE
[FEATURE] possibility to set replicasNR per volume not globally

### DISCUSSION THREAD

**Discussion by ammirator-administrator** [longhorn/longhorn#8571] [General]
What I'm interested in is if there is a possibility to set volume replicasNR at the volume level, not a global value
I know in the helm chart there is a global value for that
But can we do that per volume and is there such possibility
And if not then we can consider that s feature request

**Comment by derekbit** [comment]
You can set `volume.spec.numberOfReplicas`.
